src/v/cluster/node/local_monitor.cc

```diff
@@ -31,6 +32,9 @@
 namespace cluster::node {
 
 ss::future<> local_monitor::update_state() {
+    refresh_configuration();
```

Comment:
Can avoid polling config options here: get notifications on changes via a bind() on a property to get a config_binding, then watch() on the binding to register a callback.